src/workerd/api/node/tests/net-nodejs-test.js

```diff
@@ -437,19 +437,6 @@ export const testNetConnectImmediateFinish = {
   },
 };
 
-// test/parallel/test-net-connect-keepalive.js
-// test/parallel/test-net-keepalive.js
-// We don't actually support keep alive so this test does
-// something different than the original Node.js test
-export const testNetConnectKeepAlive = {
-  async test() {
-    throws(() => new net.Socket({ keepAlive: true }));
-    const c = new net.Socket();
-    c.setKeepAlive(false);
-    throws(() => c.setKeepAlive(true));
```

Comment:
maybe keep the test with this line commented out and replace with `c.setKeepAlive(true)` (and update the test comment)

Reply:
There is no need. There is another test that covers the existence of this function.

Reply:
Yeah what @vicb said.